src/cargo/util/config.rs

```diff
@@ -49,11 +49,16 @@ pub struct Config {
     /// The location of the rustdoc executable
     rustdoc: LazyCell<PathBuf>,
     /// Whether we are printing extra verbose messages
-    extra_verbose: Cell<bool>,
-    frozen: Cell<bool>,
-    locked: Cell<bool>,
+    extra_verbose: bool,
+    /// TODO to do with Cargo.lock
```

Comment:
This has to do with the `--frozen` flag being passed on the command line, namely we're not allowed to touch the network